utils/utils.go

```diff
@@ -119,6 +120,23 @@ func GuardedRemoveAll(path string) error {
 	return os.RemoveAll(path)
 }
 
+// RemoveFilesExcept removes all files in a directory except for the one specified
+// by excludeFile.
+func RemoveFilesExcept(path string, excludeFile string) error {
+	files, err := os.ReadDir(path)
+	if err != nil {
+		return err
+	}
+	for _, file := range files {
+		if file.Name() != excludeFile {
+			if err := os.RemoveAll(filepath.Join(path, file.Name())); err != nil {
```

Comment:
Shouldn't you use GuardedRemoveAll here for safety?

Reply:
It is not needed because iteration over the contents of the directory is performed. However, the same check could be used to make sure it doesn't remove the contents of the `/` directory.